fearless/Modules/Send/SendProtocols.swift

```diff
@@ -18,6 +18,8 @@ protocol SendViewInput: ControllerBackedProtocol, LoadableViewProtocol {
     func didBlockUserInteractive(isUserInteractiveAmount: Bool)
     func setInputAccessoryView(visible: Bool)
     func setHistoryButton(isVisible: Bool)
+    func enableSendAll()
```

Comment:
let's think about sendAll property like about viewModel. it seems more accurate and beautiful (IMO):
func didReceive(sendAllEnabled: Bool)

Reply:
I'm not sure, it's just reaction when user set enable from alert, it's always true